medusa/server/api/v2/show.py

```diff
@@ -86,6 +86,8 @@ def _handle_detailed_show(self, tv_show, query):
                 new, existing = tv_show.get_backlogged_episodes(allowed_qualities=allowed_qualities,
                                                                 preferred_qualities=preferred_qualities)
                 data = {'new': new, 'existing': existing}
+            elif query == 'archiveEpisodes':
```

Comment:
No no no. This is for GET routes. GET means retrieve data. You need to add this to the POST, PUT or PATCH as they're for adding and changing data.